src/main/java/cws/k8s/scheduler/scheduler/NodeLabelAssign.java

```diff
@@ -0,0 +1,97 @@
+package cws.k8s.scheduler.scheduler;
+
+import cws.k8s.scheduler.model.*;
+import cws.k8s.scheduler.scheduler.prioritize.Prioritize;
+import cws.k8s.scheduler.scheduler.prioritize.RankMaxPrioritize;
+import cws.k8s.scheduler.client.Informable;
+import cws.k8s.scheduler.client.KubernetesClient;
+import cws.k8s.scheduler.scheduler.nodeassign.LabelAssign;
+import cws.k8s.scheduler.scheduler.nodeassign.NodeAssign;
+import cws.k8s.scheduler.scheduler.nodeassign.FairAssign;
+import cws.k8s.scheduler.util.NodeTaskAlignment;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.*;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class NodeLabelAssign extends Scheduler {
+
+    private final Prioritize prioritize;
+    private final NodeAssign nodeAssigner;
```

Comment:
Maybe name this defaultNodeAssigner or fallbackNodeAssigner to clearify its function